docs/user/DockerUsage.md

```diff
@@ -95,6 +97,21 @@ Any number of tasks can be run at the same time. An action can run against a sin
     viya4-deployment --tags "baseline,viya,install"
   ```
 
+- I have a new cluster, deployed in OCI using https://github.com/oracle-quickstart/oke-sas-viya-public project, and I want to install baseline and deploy viya only
+- I run as non-root user
+
+  ```bash
+  sudo docker run --rm -t \
+    --group-add root \
+    --user $(id -u):$(id -g)  \
+    --volume $HOME/repos:/data:Z  \
+    --volume $HOME/repos/viya4-deployment/ansible-vars.yaml:/config/config:Z \
+    --volume $HOME/repos/oke-sas-viya-public/terraform.tfstate:/config/tfstate:Z  \
+    --volume $HOME/.ssh/id_rsa:/config/jump_svr_private_key:z  \
+    --volume $HOME/repos/viya4-deployment/.oci:/viya4-deployment/.oci:Z \
+    viya4-deployment --tags "baseline,viya,install"  
+  ```
+
```

Comment:
Can you explain the :Z you have at the end of each of these lines. Thx

Reply:
I did a lot of debugging of docker run and found the volumes specified on the command line were not accessible inside the container until I added :Z at the end. I found this solution on the internet, and it might be related to running sudo docker run.

Reply:
Is your docker image based on Ubuntu 20.04 from our main repo or have you converted these to Fedora or some other Red Hat flavor?

Reply:
Here is the Docker file I used
https://github.com/oracle-quickstart/oke-sas-viya-public/blob/master/Dockerfile

Reply:
These still need to be removed.

Reply:
If you mean :Z at the end of --volume line is to be removed does it mean you want users to run into the same troubles as I did and not to have any potential guidance on how to solve it ?